docs/controlling-fixtures/using-the-select-buttons-and-wheels.md

```diff
@@ -108,25 +108,41 @@ change, then press \<Record\>, then \<Locate\>. Select \[Shared\] or
 \[Individual\] from the softkeys. Press \<Record\> or \<Locate\> a
 second time to confirm.
 
-## Clearing Selection
+## Clearing the fixture selection and the programmer
 
 The \<Clear\> button (on the right of the numeric keypad) is used to
-remove all changes from the Programmer and deselect all fixtures. A
-quick press of the \<Clear\> button just clears everything, however if
-you hold down the \<Clear\> button, then more options are available.
-
--   You can mask which attributes are to be cleared (for example,
-    leaving the pan/tilt in the programmer but clearing everything else)
-    by holding down \<Clear\> and pressing \[Set Mask to Clear
-    Nothing\]. Then (still holding \<Clear\>) turn on the Attributes you
-    want to change using the Attribute Bank buttons down the right hand
-    side, or press \[Set Mask\] and use the softkeys. Only the lit
-    attributes will be cleared. Pressing the \<Options\>/\<Attribute
-    Options\> button will clear the mask. The Time mask option allows
-    you to clear or retain the fade/delay times in the programmer for
-    all attributes (clearing individual attributes will also clear times
-    for that attribute, for example setting a P mask will clear position
-    times).
+remove all changes from the programmer and deselect all fixtures. 
+
+Normally a **quick press** of the \<Clear\> button 
+clears the fixture selection and also clears from the programmer all the
+fixture changes you've made. 
+However you can change this so that **double presses** of the button will clear
+only the fixture selection on the first press, then the programmer on the
+second press, or the other way round. See [Action Precedence](using-the-select-buttons-and-wheels.md#clear-options-menu)
+below in the Clear Options menu.
+
+### Clear button hold-down options
+If you hold down the \<Clear\> button, then more options are available on
+the softkeys. These will take effect when you release the \<Clear\> button.
+
+The **Set Mask** options let you choose which attribute groups are to be cleared (for example,
```

Comment:
Might be worth highlighting that you can also directly hit an attribute group to modify the mask